packages/eslint-config/index.js

```diff
@@ -1,3 +1,52 @@
 module.exports = {
-  extends: 'airbnb-base',
-};
+    extends: 'airbnb-base',
+    rules: {
+        'arrow-parens': ['error', 'as-needed'],
+        'comma-dangle': ['error', 'only-multiline'],
+        'eol-last': 'off',
+        /*
+            This is the default value which is defined here to note that there may be exceptions
+            where it is appropriate to disable this rule for older projects.
+        */
+        'func-names': ['error', 'as-needed'],
+        indent: ['error', 4],
+        'linebreak-style': 'off',
+        /*
+            This is the default value which is defined here to note that there may be exceptions
+            where it is appropriate to disable this rule for older projects or with a comment.
```

Comment:
I propose changing "with a comment" to "in specific files" just to keep the focus on the "what" instead of the "how".

Reply:
This is intentional. The distinction between projects and comments is to convey specifically whether the exception should be per instance (file or line) or generally. In this sense, I am including the "how" to provide additional information.

Reply:
I see. I'm probably nitpicking, but "with a comment" comes across a bit vague to me. Other suggestions: 
1. "with a file suppression comment" (I'm assuming this would always be disabled per-file...if not then "with a file or line suppression comment")
1. "in specific files with `// tslint:disable:max-classes-per-file`"

Reply:
I would be prefer to refer the reader to the ESLint documentation for the correct syntax, so I have gone with, "or by suppressing with a file or inline comment."